Library/Homebrew/cmd/missing.rb

```diff
@@ -6,15 +6,41 @@ module Homebrew
   def missing
     return unless HOMEBREW_CELLAR.exist?
 
-    ff = if ARGV.named.empty?
+    formulae = if ARGV.named.empty?
       Formula.installed
     else
       ARGV.resolved_formulae
     end
+    if ARGV.json == "v1"
+      missing = print_missing_json(formulae)
+    else
+      missing = print_missing(formulae)
```

Comment:
In Ruby it is preferable to assign the return value of the `if` expression instead of having separate assignments in its various branches (see the `if ARGV.named.empty?` directly above).